src/docdb/tree/DocDBStoredProcedureTreeItem.ts

```diff
@@ -15,7 +18,7 @@ export class DocDBStoredProcedureTreeItem implements IAzureTreeItem {
     public static contextValue: string = "cosmosDBStoredProcedure";
     public readonly contextValue: string = DocDBStoredProcedureTreeItem.contextValue;
 
-    constructor(private _procedure: ProcedureMeta) {
+    constructor(private _documentEndpoint: string, private _masterKey: string, private _isEmulator: boolean, private _procedure: ProcedureMeta) {
```

Comment:
1. Do procedures have a unique endpoint? If yes, I prefer calling it something like `storedProcEndpoint`, `ProcedureEnpoint`, or something to that effect. :) 
2. Can the variables masterKey, and isEmulator be inferred from the parent?

Reply:
Might make sense, but not going to worry about it now.